.gitignore

```diff
@@ -117,4 +117,12 @@ env/
 .venv/
 
 ### Configuration files ###
-.env
\ No newline at end of file
+.env
+
+### Terraform
+.terraform
+.terraform.lock.hcl
```

Comment:
It's worth noting that this file is a little like a package-lock.json or poetry.lock, and thus _should_ be checked in [as per the terraform docs](https://www.terraform.io/docs/language/dependency-lock.html#lock-file-location)